mtp_bank_admin/apps/bank_admin/__init__.py

```diff
@@ -0,0 +1,3 @@
+ADI_PAYMENT_LABEL = 'ADIPAYMENT'
+ADI_REFUND_LABEL = 'ADIREFUND'
+ACCESSPAY_LABEL = 'ACCESSPAY'
```

Comment:
I wonder if these might be better as `RECONCILED_PAYMENTS`, `RECONCILED_REFUNDS`, and `ISSUED_REFUNDS`, or something along those lines? Certainly `ACCESSPAY` gives no indication of the direction of the funds.